powerline/selectors/tmux.py

```diff
@@ -0,0 +1,34 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+from powerline.bindings.tmux import get_tmux_output
+
+
+_cached_clients = {}
+
+def tmux_via_ssh(pl, segment_info, mode):
+	'''Returns True if tmux client is connected via SSH
+	
+	Requires the ``psutil`` module.
+	
+	Works only for tmux versions >= 2.1'''
```

Comment:
This is tied to `--process-id` being supplied on command-line, not to tmux version in any fashion. Also closing `'''` needs to be on its own line and it is not qualified to be a sentence end.

Reply:
#{client_pid} got introduced with tmux 2.1, so it is not supported before

Reply:
This is tmux bindings implementation detail. All what *selector* cares about is whether `--process-id` is supplied or not, if it is it would be completely happy without even having tmux installed.